xbmc/windows/GUIMediaWindow.cpp

```diff
@@ -727,30 +713,36 @@ bool CGUIMediaWindow::GetDirectory(const std::string &strDirectory, CFileItemLis
 // \brief Set window to a specific directory
 // \param strDirectory The directory to be displayed in list/thumb control
```

Comment:
This would also need to be ```_arg``` I guess? And the second param is missing, but that's not your doing :/